images/pyspark-notebook/Dockerfile

```diff
@@ -68,6 +68,6 @@ RUN mamba install --yes \
     mamba clean --all -f -y && \
     fix-permissions "${CONDA_DIR}" && \
     fix-permissions "/home/${NB_USER}"
-
+RUN pip install pyspark
```

Comment:
Can we use `mamba` here?

Also, this is not how we install things.
Please, copy-paste the installation method we're using from any other our image, because we don't want to pollute our images with caches and so on.